guides/source/layouts_and_rendering.md

```diff
@@ -299,12 +299,14 @@ Calls to the `render` method generally accept five options:
 
 ##### The `:content_type` Option
 
-By default, Rails will serve the results of a rendering operation with the MIME content-type of `text/html` (or `application/json` if you use the `:json` option, or `application/xml` for the `:xml` option.). There are times when you might like to change this, and you can do so by setting the `:content_type` option:
```

Comment:
There is no reason to mention about all of the content-type, but it is worth especially if `application/json` to me.